### ISSUE TITLE
Will u release training code ? 

### ISSUE THREAD

**Issue by HB-pencil-zero** [a1600012888/PhysDreamer#24]
Hello, thanks ur good work. I wonder when will u release ur code ? Thank you

**Comment by a1600012888** [comment]
The material field training is in `projects/uncleaned_train/exp_motion/train/train_material.py`

The velocity field t raining is in `projects/uncleaned_train/exp_motion/train/fast_train_velocity.py`

For the auto-diff part of MPM(with gradient checkpointing), please check `projects/uncleaned_train/exp_motion/train/interface.py`

Some details to be careful about:
1. About the point downsampling for the 3DGS,  I print up the average, max points per grid after downsampling. My experience is that average point per grid around 10 particles is enough. 
2. For velo training, you need early stop. My experience is around 100-200 train iters is enough. Otherwise using single frame video would overfit the motions.
3. When tuning hyper-param, please check the PSNR of the simulated video and original videos. My experience is that it typically goes up to 29-30 after training.
